test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -9600,7 +9600,7 @@ def import_hook_config(self, hook_name, hook_conf, hook_type,
         else:
             hook_t = PBS_HOOK
 
-        fn = self.du.create_temp_file(body=hook_conf)
+        fn = self.du.create_temp_file(body=json.dumps(hook_conf))
```

Comment:
Optional: I suggest to pass indent=2 for it will dump JSON in a pretty format, will be easy to read later (mostly when it get saved in post-analysis data)